src/Console/Commands/FluxOptimize.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace FluxErp\Console\Commands;
+
+use FluxErp\Facades\Action;
+use FluxErp\Facades\Repeatable;
+use FluxErp\Traits\HasDefault;
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\Relation;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Str;
+
+class FluxOptimize extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'flux:optimize';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Warms the cache for the Flux application';
+
+    protected bool $forget = false;
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): void
+    {
+        $this->optimizeDefaultModels();
+        $this->optimizeModelInfo();
+
+        if ($this->forget) {
+            $this->optimizeActions();
+            $this->optimizeViewClasses();
+            $this->optimizeRepeatable();
+        }
+    }
+
+    protected function optimizeDefaultModels(): void
+    {
+        foreach (Relation::morphMap() as $alias => $model) {
+            if (! in_array(HasDefault::class, class_uses_recursive($model))) {
+                continue;
+            }
+
+            // query the default model to warm the cache
```

Comment:
this comment is only partially correct. If $this->forget = true the cache is not warmed.